BnbnavNetClient/Models/Edge.cs

```diff
@@ -32,4 +32,15 @@ public void Deconstruct(out string Id, out Road Road, out Node From, out Node To
         From = this.From;
         To = this.To;
     }
+
+    public bool CanSnapTo()
+    {
+        return Road.RoadType != RoadType.DuongWarp;
```

Comment:
Why is this not a property?

Reply:
Lift+shift from C++ - I'll change it to a property